f8a_report/helpers/report_helper.py

```diff
@@ -64,32 +64,34 @@ def __init__(self):
 
         self.emr_api = os.getenv('EMR_API', 'http://f8a-emr-deployment:6006')
 
+    def cleanup_tables(self, table_name, env_var, num_days):
+        """Cleanup tables on a periodic basis."""
+        # Number of days to retain the celery task_meta data
+        num_days_metadata = os.environ.get(env_var, num_days)
+        # Create query string
+        delete_str = 'DELETE FROM ' + table_name + 'WHERE DATE_DONE <= NOW() - interval \'%s day\';'
+        # query to delete data
+        query = sql.SQL(delete_str)
+        logger.debug('Starting to clean up ' + table_name + 'tables')
```

Comment:
use log native string formatting.
```suggestion
        logger.debug('Starting to clean up "%s" table', table_name)
```